openedx/core/djangoapps/programs/tests/test_tasks.py

```diff
@@ -0,0 +1,244 @@
+"""
+Tests for programs celery tasks.
+"""
+
+import ddt
+from django.test import TestCase
+from edx_rest_api_client.client import EdxRestApiClient
+import httpretty
+import json
+import mock
+
+from oauth2_provider.tests.factories import ClientFactory
+from student.tests.factories import UserFactory
+
+from openedx.core.djangoapps.credentials.tests.mixins import CredentialsApiConfigMixin
+from openedx.core.djangoapps.programs.tests.mixins import ProgramsApiConfigMixin
+from openedx.core.djangoapps.programs import tasks
```

Comment:
i've occasionally seen (and used) the approach of further subdividing first-party imports to separate the package under test from other supporting packages.   i guess that may be something i dreamed / invented rather than an actual practice, so i'm changing this as requested.